packages/app/source/client/controllers/todos-controller.js

```diff
@@ -6,13 +6,18 @@ Space.Object.extend(Todos, 'TodosController', {
   ],
 
   dependencies: {
-    configuration: 'configuration'
+    configuration: 'configuration',
+    todosStore: 'Todos.TodosStore',
   },
 
   eventSubscriptions() {
     return [{
       'Todos.TodoCreated': this._onTodoCreated,
       'Todos.TodoToggled': this._onTodoToggled,
```

Comment:
Oh I didn't notice this. Toggling is a component concept IMO. It should determine the state change using the store dependency, then broadcast the change